src/svg.resize.js

```diff
@@ -51,21 +49,26 @@ class ResizeHandler {
       return
     }
 
+    this.box = this.el.bbox()
+    this.startPoint = this.el.point(startX, startY)
```

Comment:
you dont need to run this functions when the event gets preventen in the next line. So move them under the event check to potentially save time